libs/coin-modules/coin-tezos/src/api/index.test.ts

```diff
@@ -118,14 +123,14 @@ describe("Testing estimateFees function", () => {
 
   it("should return estimation from logic module", async () => {
     logicEstimateFees.mockResolvedValue({ estimatedFees: DEFAULT_ESTIMATED_FEES });
-    const result = await api.estimateFees({ type: "send" } as TransactionIntent<TezosAsset>);
-    expect(result).toEqual(DEFAULT_ESTIMATED_FEES);
+    const result = await api.estimateFees({ type: "send", sender: {} } as TezosTransactionIntent);
+    expect(result.value).toEqual(DEFAULT_ESTIMATED_FEES);
```

Comment:
The assertion has been weakened here: we moved from a global equality to an equality restricted to a single property.
```suggestion
    expect(result).toEqual({ value: DEFAULT_ESTIMATED_FEES });
```
We ensure there is no extra keys (this could be applied to the following tests as well)